docs/LifMarketValidationMechanism.md

```diff
@@ -0,0 +1,390 @@
+* [LifMarketValidationMechanism](#lifmarketvalidationmechanism)
+  * [getBuyPrice](#function-getbuyprice)
+  * [totalPausedSeconds](#function-totalpausedseconds)
+  * [lifToken](#function-liftoken)
+  * [getMaxClaimableWeiAmount](#function-getmaxclaimableweiamount)
+  * [initialBuyPrice](#function-initialbuyprice)
+  * [unpause](#function-unpause)
+  * [secondsPerPeriod](#function-secondsperperiod)
+  * [initialWei](#function-initialwei)
+  * [totalBurnedTokens](#function-totalburnedtokens)
+  * [paused](#function-paused)
+  * [changeToken](#function-changetoken)
+  * [getAccumulatedDistributionPercentage](#function-getaccumulateddistributionpercentage)
+  * [isFinished](#function-isfinished)
+  * [pause](#function-pause)
+  * [owner](#function-owner)
+  * [pausedTimestamp](#function-pausedtimestamp)
+  * [totalReimbursedWei](#function-totalreimbursedwei)
+  * [totalWeiClaimed](#function-totalweiclaimed)
+  * [getCurrentPeriodIndex](#function-getcurrentperiodindex)
+  * [fund](#function-fund)
+  * [calculateDistributionPeriods](#function-calculatedistributionperiods)
+  * [claimWei](#function-claimwei)
+  * [originalTotalSupply](#function-originaltotalsupply)
+  * [startTimestamp](#function-starttimestamp)
+  * [periods](#function-periods)
+  * [transferOwnership](#function-transferownership)
+  * [funded](#function-funded)
+  * [sendTokens](#function-sendtokens)
+  * [totalPeriods](#function-totalperiods)
+  * [foundationAddr](#function-foundationaddr)
+  * [Pause](#event-pause)
+  * [Unpause](#event-unpause)
+  * [ClaimedWei](#event-claimedwei)
+  * [SentTokens](#event-senttokens)
+  * [OwnershipTransferred](#event-ownershiptransferred)
+
+# LifMarketValidationMechanism
+
+
+## *function* getBuyPrice
+
+LifMarketValidationMechanism.getBuyPrice() `view` `018a25e8`
+
+> returns the current buy price at which the MVM offers to buy tokens to burn them
+
+
+
+Outputs
+
+| **type** | **name** | **description** |
+|-|-|-|
+| *uint256* | price | undefined |
```

Comment:
All the outputs can be documented in a nicer way if we use the JSON notation for named outputs, in this case:

```
     @return  { "price": "the current buy price (in eth/lif, multiplied by PRICE_FACTOR)"}
    */
  function getBuyPrice() public view returns (uint256 price) {
```

Such changes in natspec docs will also resolve solmd warnings that occur during `npm run soldoc`

Reply:
awsm, thx @JirkaChadima